flask_rbac/__init__.py

```diff
@@ -256,7 +257,7 @@ def a_view_func():
             roles = _user.get_roles()
         return self._check_permission(roles, method, endpoint)
 
-    def allow(self, roles, methods, with_children=True):
+    def allow(self, roles, methods, with_children=True, endpoint=None):
```

Comment:
Please add some use cases of using `endpoint` (e.g. in `docs/quickstart.rst.inc`) , for helping others understand the usage of this argument.